### ISSUE TITLE
Auto-generate default plots

### ISSUE THREAD

**Issue by ryanbanderson** [USGS-Astrogeology/PySAT_Point_Spectra_GUI#153]
For several modules, there are default plots that you basically always want to produce. We could potentially streamline the user experience by just automatically producing these.

- [ ] Histograms for folds. See #152  
- [ ] RMSECV and RMSEC vs # of components for PLS cross validation
- [ ] RMSECV vs # of components for GP cross validation
- [ ] RMSECV vs Alpha (log scale) for Lasso and Elastic Net cross validation (must handle permutations of intercept and positive coefficients)
- [ ] One to one plot of training data for regression train. (Could optionally also plot predictions for a test set)
